app/views/admins/show_procedures.html.erb

```diff
@@ -1,12 +1,11 @@
-All Procedures | <%= link_to "Pending Procedures", admins_path, method: :'get' %>
+All Procedures | <%= link_to "Back To Admin Page", admins_path() %>
 
 <div>
   <% @procedures.each do |procedure|%>
     <div>
-      <%=procedure.name%> <%= procedure.date %> <%= procedure.description %> <%= procedure.price %> 
-          <%= procedure.status %> <%= procedure.facility_id %>
-      <%= link_to "Edit", edit_procedure_path(procedure.id), method: :'get' %>
-      <%= link_to "Delete", procedure_path(procedure.id), method: :'delete' %>
+    <%=procedure.name%> <%= procedure.description %>
+    <%= link_to "Edit", edit_procedure_path(procedure.id), method: :'get' %>
```

Comment:
You don't need to specify the method here since `get` is the default for this path.

```suggestion
    <%= link_to "Edit", edit_procedure_path(procedure) %>
```

Reply:
removed method: :get